dev/io.openliberty.cdi.core_fat/fat/src/io/openliberty/cdi/internal/core/interceptors/CDIInterceptorTest.java

```diff
@@ -0,0 +1,48 @@
+/*******************************************************************************
+ * Copyright (c) 2022 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package io.openliberty.cdi.internal.core.interceptors;
+
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.spec.WebArchive;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.runner.RunWith;
+
+import componenttest.annotation.TestServlet;
+import componenttest.custom.junit.runner.FATRunner;
+import componenttest.topology.impl.LibertyServer;
+import io.openliberty.cdi.internal.core.FATSuite;
+import io.openliberty.cdi.internal.core.interceptors.app.CDIInterceptorTestServlet;
+
+@RunWith(FATRunner.class)
+public class CDIInterceptorTest {
+
+    @TestServlet(contextRoot = "cdiInterceptor", servlet = CDIInterceptorTestServlet.class)
+    public static LibertyServer server;
+
+    @BeforeClass
+    public static void setup() throws Exception {
+        server = FATSuite.server;
+
+        Package appPackage = CDIInterceptorTestServlet.class.getPackage();
+        WebArchive war = ShrinkWrap.create(WebArchive.class, "cdiInterceptor.war")
+                                   .addPackage(appPackage)
+                                   .addAsWebInfResource(appPackage, "beans.xml", "beans.xml");
```

Comment:
delete this one

Reply:
Yes, if we decide not to include a beans.xml, I will remove it from all apps.